### ISSUE TITLE
Improvement Suggestion - Permission Control

### ISSUE THREAD

**Issue by geanferrani123** [Ylianst/MeshCentral#595]
Ylianst, good morning!

Is it possible to create a permission option to release all remote connection features (chat, terminal, remote machine files, etc.) automatically after the user accepts the remote connection on their desktop? For some environments, this option would be very interesting because it would eliminate the need for the user to authorize everything, because if you are logged in the machine, you optimize your time to use the other features that facilitated the service.

**Comment by Ylianst** [comment]
Yes, I will be adding this. Just need time to get it done. Note that I will be traveling for the first two weeks of November and so, expect to only fix critical things during that time.

**Comment by MailYouLater** [comment]
Duplicate of #584.

**Comment by Ylianst** [comment]
@MailYouLater is correct, this is a duplicate by the same author. Should we not close this one?

**Comment by geanferrani123** [comment]
Please close 584, I think it's better to keep this one open
